app/assets/stylesheets/question_form.scss

```diff
@@ -0,0 +1,6 @@
+@import "colors";
+@import "foundation_and_overrides";
+
+.question-form-row {
+ display:inline;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Colon after property should be followed by one space